src/components/z-navigation-tabs/styles.css

```diff
@@ -134,7 +134,7 @@ z-navigation-tabs > nav > a {
 
 z-navigation-tabs > nav > *:focus:focus-visible {
   z-index: 1;
-  box-shadow: inset 0 0 4px 3px var(--color-highlight);
+  box-shadow: inset 0 0 4px 3px var(--blue800);
```

Comment:
Come mai questo cambiamento? Penso sia più corretto mantenere `--color-highlight` che è temizzato, no?

Reply:
Avevamo chiesto a Francesco, e lo abbiamo cambiato perchè pensavamo fosse il vecchio focus, però se pensi fosse meglio prima lo rimettiamo a posto

Reply:
`--color-highlight` al momento è un `--blue400`. Quando ho aggiornato i token del focus non ho modificato `--color-highlight` perché non so esattamente quale uso se ne faccia all'esterno. Mentre all'interno del design system questo è l'unico punto in cui quel token viene utilizzato.